landlab/utils/fileTreatment.py

```diff
@@ -0,0 +1,9 @@
+import os
+
+
+def check_out_dirs(out_file_dir):
```

Comment:
I use the function outside Landlab e.g. in various notebooks to check directories before storing figures/files related to Landlab calculations. Does that legitimize a function in Landlab or is it better to keep those things outside?

Reply:
It seems like it might be outside the scope of *landlab*. In your notebooks, instead of creating a function, you could use,
```python
os.path.isdir(out_file_dir) or os.makedirs(out_file_dir)
```

Reply:
Fair enough. It's gone